setup.py

```diff
@@ -368,13 +368,6 @@ def setup_package():
         cmdclass={"sdist": sdist_checked},
         python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
         zip_safe=False,
-        entry_points={
-            'console_scripts': [
-                'f2py = numpy.f2py.__main__:main',
-                'conv-template = numpy.distutils.conv_template:main',
-                'from-template = numpy.distutils.from_template:main',
```

Comment:
I think these last two lines can stay? Were `conv-template` and `from-template` on the path before?

I suppose the other way of looking at it is that these are awfully generic names to add to the user's path.

Reply:
They are only used during builds and, AFAICT, only other functions in `numpy/distutils` use them, so I don't see the need to include them in a bin directory or windows equivalent. Note that the names are also changed.

The reason there are two such functions is that f2py and numpy have their own separate template definitions. It would be nice if some day we only needed one :)

Reply:
Which is also probably sufficient reason to remove the `main` functions that were added in #10463 and just do a complete reversion, but those added functions are also harmless.